modules/manageBook.js

```diff
@@ -0,0 +1,65 @@
+import Book from './bookClass.js';
+
+export default class ManageBookDetails {
+  constructor() {
+    this.books = [];
+  }
+
+  displayBooks = () => {
+    const bookTableBody = document.getElementById('bookTableBody');
+    bookTableBody.innerHTML = '';
+
+    this.books.forEach((book, index) => {
+      const row = document.createElement('tr');
+
+      const titleCell = document.createElement('td');
+      titleCell.textContent = book.title;
+      row.appendChild(titleCell);
+
+      const authorCell = document.createElement('td');
+      authorCell.textContent = book.author;
+      row.appendChild(authorCell);
+
+      const removeCell = document.createElement('td');
+      const removeButton = document.createElement('button');
+      removeButton.className = 'remove-button';
+      removeButton.textContent = 'Remove';
+      removeButton.addEventListener('click', () => {
+        this.removeBook(index);
+      });
+      removeCell.appendChild(removeButton);
+      row.appendChild(removeCell);
+
+      bookTableBody.appendChild(row);
+    });
+  }
```

Comment:
- [ ] 👍 Well! Your books are displaying perfectly when I click on the `List` nav item. However, there are three content sections in the wireframe: Books list, Add book form, and Contact info. Currently, you are showing only the title of the "books list" section by default. I recommend showing the table with the section title by default when there is at least one book, as per the project requirement. This way, users can see all the books instantly without having to click the "List" nav item. 📚📝

[Awesome-Books--JavaScript-Project.webm](https://github.com/Armynerh/awesome_books-es6/assets/55840999/28ad0414-3d0c-4df1-a39b-2de721303f0a)